src/util/os_utils.js

```diff
@@ -995,6 +1041,7 @@ exports.read_drives = read_drives;
 exports.get_raw_storage = get_raw_storage;
 exports.remove_linux_readonly_drives = remove_linux_readonly_drives;
 exports.get_main_drive_name = get_main_drive_name;
+exports._df = _df;
```

Comment:
@liranmauda why are you exporting a function with _ prefix? this is quite confusing....

Reply:
@guymguym I agree, It is a backport of @nadavMiz code, 
@nadavMiz why are you exporting a function with _ prefix? this is quite confusing....

Reply:
Nadav is OOO and won't be able to answer